src/components/Checkout/TeamAttribution.vue

```diff
@@ -101,7 +101,7 @@ export default {
 							this.selectedId === 0 ? 'Team Attribution Removed'
 								: 'Team Attribution Removal Success', numeral(this.selectedId).value()
 						);
-						this.$emit('refresh-totals', 'team-update');
+						this.$emit('refreshtotals', 'team-update');
```

Comment:
From what I can see, this won't do anything. The TeamAttribution.vue component only lives in the BasketItem.vue file and the component usage doesn't handle any emitted events at this time. You would need to add the `@refreshtotals="doSomething"` to `<team-attribution>`.

Reply:
I guess I was just trying to be thorough and make sure every instance of event emission related to refresh totals is without the dash.  Do I need to revert or can I just let this be?

Reply:
done